README.md

```diff
@@ -192,13 +195,17 @@ df = pd.read_csv("my_dataset.csv")
 #### Modin is a DataFrame designed for datasets from 1MB to 1TB+ 
 
 We have focused heavily on bridging the solutions between DataFrames for small data 
-(e.g. pandas) and large data. Often data scientists require different tools for doing
-the same thing on different sizes of data. The DataFrame solutions that exist for 1KB do
-not scale to 1TB+, and the overheads of the solutions for 1TB+ are too costly for 
-datasets in the 1KB range. With Modin, because of its light-weight, robust, and scalable
-nature, you get a fast DataFrame at small and large data. With preliminary [cluster](https://modin.readthedocs.io/en/latest/using_modin.html#using-modin-on-a-cluster)
-and [out of core](https://modin.readthedocs.io/en/latest/out_of_core.html)
-support, Modin is a DataFrame library with great single-node performance and high
+(e.g. pandas) and large data. Often data scientists have to use different tools 
+for operating on datasets of different sizes. Processing large dataframes with pandas
+is slow, and pandas does not support working with dataframes that are too large to fit 
+into the available memory. As a result, pandas workflows that work well 
+for prototyping on a few MBs of data do not scale to tens or hundreds of GBs (depending on the size 
+of your machine). Modin supports operating on data that does not fit in memory, so that you can comfortably 
+work with hundreds of GBs without worrying about substantial slowdown or memory errors. 
+
+With preliminary [cluster](https://modin.readthedocs.io/en/latest/getting_started/using_modin.html?#using-modin-on-a-cluster-experimental)
```

Comment:
I would suggest replacing this link to cluster usage to using_modin_cluster page after #3965 is merged.

Reply:
+1, it is merged now